### ISSUE TITLE
ImportError: validation_error_cause flag requires the exceptiongroup module backport to be installed when used on Python <3.11.

### ISSUE THREAD

**Issue by VinayChaudhari1996** [sinaatalay/RenderCV#66]
**Describe the bug**
Getting error after : ```rendercv render FILE.json```

Python Version:
> 3.10.11

> It was working fine with previous version

### Error 
![image](https://github.com/sinaatalay/rendercv/assets/42869040/77b32861-1e34-40ed-bf3f-c0eb44964be1)

**Comment by averagejoey2000** [comment]
I also get this error when using `rendercv render Full_Name_CV.yaml`
John Doe's resume compiles, so I'm picking through mine to try to figure out what syntax error I made that stopped it working

**Comment by sinaatalay** [comment]
@averagejoey2000 Could you please share the YAML file and your Python version?

**Comment by averagejoey2000** [comment]
So I was up late last night and was able to fix the error

My python is Python 3.10.12
if it helps, that's WSL Ubuntu with `python3/jammy-updates,jammy-security,now 3.10.6-1~22.04 amd64`

Of course, now going back through my commits I am finding that half of my git history where it was broken is no longer giving me this error, so I had to go back further than I thought.

hmm. Github doesn't allow me to upload a yaml file.
The error does occur in this version
https://github.com/averagejoey2000/resume/blob/128f16396bd00a48348b2aece80ce9b08a5c7942/Joey_Simone_CV.yaml

The changes in [this commit](https://github.com/averagejoey2000/resume/commit/26fd667d85463f06aef4bb942832452847c51313) fix the error and the document compiles perfectly. However, I had to make all those changes by hand, without help from the error checking system, based on what I learned up late last night.

It would have been nice if the program gave me a usable error message with guidance, like these messages I got later on 
![at symbol mandatory](https://github.com/sinaatalay/rendercv/assets/7506195/47eb19a3-0c5c-4c74-899e-346d3d078229)
![at symbol not allowed](https://github.com/sinaatalay/rendercv/assets/7506195/8cb9e32a-72b8-487a-a9c8-c837cb92ac83)
